include/ctrlc.h

```diff
@@ -34,18 +34,18 @@ bool utSetInterruptEnabled(bool);
 /**
  * Start listner for ctrl-c interrupts
  */
-void startInterruptListener(void);
```

Comment:
Can you add a namespace for the .h/.cc files?

Reply:
OSQP is written in C not C++. As far as I know, C does not have namespaces. Are you having any issues with conflicting function names? Please file a separate issue for that.